include/CheatingRockMuonTaggingTool.h

```diff
@@ -0,0 +1,38 @@
+/**
+ *  @file   include/CheatingRockMuonTaggingTool.h
+ *
+ *  @brief  Header file for the cheating rock muon tagging tool class.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_CHEATING_ROCK_MUON_TAGGING_TOOL_H
+#define LAR_CHEATING_ROCK_MUON_TAGGING_TOOL_H 1
+
+#include "larpandoracontent/LArControlFlow/CosmicRayTaggingBaseTool.h"
+#include "larpandoracontent/LArControlFlow/MasterAlgorithm.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  CheatingRockMuonTaggingTool class
+ */
+class CheatingRockMuonTaggingTool : public CosmicRayTaggingBaseTool
```

Comment:
Maybe in future we want to rename the CosmicRayTaggingBaseTool to make it more generic, if this is doing track-like backgrounds more broadly? (Not a big consideration for now of course, and again, this would end up affecting everyone else too)